### PR TITLE
Trion Interactive 1.0 adapter update

### PR DESCRIPTION
**Pull request by grohm33**
…rebid version(1.0).

## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other

## Description of change

Adding updated adapter for the 1.0 release

- contact email of the adapter’s maintainer
mgroh@trioninteractive.com
- [ ] official adapter submission

## Other information

We have a test ad set up to return bids from pubId : '30222' and sectionId : '5' and will only do so on mobile user agent.

### CODE REVIEW

File: modules/trionBidAdapter.js [hunk 1]
```diff
@@ -1,129 +1,153 @@
...
-      }, true);
-    } else {
-      userTag = window.TRION_INT;
-      getBids(bids);
-    }
-  };
+  interpretResponse: function (trionResponseObj, request) {
+    var bid = {};
+    var bidResponses = [];
+    var bidRequest = request.bidRequest;
```
Comment (snapwich) [inline modules/trionBidAdapter.js]: Where is this coming from?  I think you're doing something similar to what I did in the rubicon adapter to get the original bidRequest, but I attached it to the request;  I don't see you doing the same in this code.

Comment (grohm33) [reply modules/trionBidAdapter.js]: Hey @snapwich, the bidrequest is included in the build request function on line 25 of the adapter.  I'm sending it up with the bid request object for each bid.  The request is coming through on all the tests on my end.  Let me know if there is something I'm missing.

File: modules/trionBidAdapter.js [hunk 2]
```diff
@@ -1,129 +1,153 @@
...
+      var result = trionResponseObj.result;
+
+      if (result && result.cpm && result.placeBid && result.ad) {
+        var cpm = parseInt(result.cpm, 10) / 100;
 
-  function getBids(bids) {
-    if (!userTag.int_t) {
-      userTag.int_t = window.TR_INT_T || -1;
+        bid.requestId = bidRequest.bidId;
+        bid.bidderCode = bidRequest.bidder;
```
Comment (mkendall07) [inline modules/trionBidAdapter.js]: This line needs to be dropped. Sorry for the confusion.

**Comment by mkendall07** [comment]
@grohm33 One thing to fix and then this can merge. Sorry for the delay!

**Comment by grohm33** [comment]
@mkendall07 the bidder code was removed.  Thanks!

**Pull request merged**